sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -0,0 +1,543 @@
+# Azure Text Analytics client library for JavaScript
```

Comment:
should we replace `Text Analytics` here and in other places where we reference the SDK by friendly name?

Reply:
I renamed this to `Azure Text Analysis client library for JavaScript`. I didn't use `Language` because this client doesn't cover all its features.